test/blocks/countdown-timer/countdown-timer.test.js

```diff
@@ -0,0 +1,26 @@
+import { readFile } from '@web/test-runner-commands';
```

Comment:
I recommend a HTML first test file like `countdown-timer.test.html` that contains the markup either directly or inside a `template` and your test file `countdown-timer.test.html.js` would actually contain the test logic (or inline the js in the html)
This would avoid you having to use `await readFile` method that is available only when you run the WTR debugger.
Also, with the template approach above, you can add plenty of tests easily without having to add more and more files in mocks folder.

we do this a lot